dojo/tools/appcheck_web_application_scanner/engines/base.py

```diff
@@ -220,10 +220,23 @@ def parse_status(self, finding: Finding, value: str) -> None:
         elif value == "acceptable_risk":
             finding.risk_accepted = True
 
+    #####
+    # For severity (extracted from cvss vector)
+    #####
+    def get_severity(self, value: str) -> str | None:
+        if cvss_obj := cvss.parser.parse_cvss_from_text(value):
+            if (severity := cvss_obj[0].severities()[0].title()) in Finding.SEVERITIES:
+                return severity
+        return None
+
+    def parse_severity(self, finding: Finding, value: str) -> None:
+        if severity := self.get_severity(value):
+            finding.severity = severity
+
```

Comment:
Looks like this was accidentally added back after it was removed [here](https://github.com/DefectDojo/django-DefectDojo/commit/86aeeffbd5747f672f72422b54f384bcd6e34a18#diff-803c93f68904dbec47d51dcf6f65060d3cac54249f5ccca020a1dc9475801e02L195) in PR #10918

Reply:
Thank. Done.